features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -0,0 +1,829 @@
+                                width={ 16 }
+                                FormControlProps={ {
+                                    margin: "dense"
+                                } }
+                                ariaLabel="username"
+                                required={ true }
+                                data-componentid={ `${ _componentId }-authentication-type-dropdown` }
+                                name="authenticationType"
+                                type={ "dropdown" }
+                                displayEmpty={ true }
+                                label={ t("console:manage.features.actions.fields.authenticationType.label") }
+                                placeholder={ t("console:manage.features.actions.fields." +
+                                    "authenticationType.placeholder") }
+                                component={ SelectFieldAdapter }
+                                maxLength={ 100 }
+                                minLength={ 0 }
+                                options={
+                                    [ ...ActionsConstants.AUTH_TYPES.map(
+                                        (option: AuthenticationTypeDropdownOption) => ({
+                                            text: t(option.text),
+                                            value: option.value.toString() }))
+                                    ]
+                                }
+                                onChange={ handleAuthTypeChange }
+                                disabled={ getFieldDisabledStatus() }
+                            />
+                            { renderAuthenticationPropertyFields() }
+                        </>
+                    );
+                };
+
+                return (
+                    <Box className="box-container">
+                        <div className="box-field">
+                            { renderAuthentication() }
+                            { !isCreateFormState && (
+                                <Button
+                                    onClick={ handleAuthenticationChangeCancel }
+                                    variant="outlined"
+                                    size="small"
+                                    className="secondary-button"
+                                    data-componentid={ `${ _componentId }-cancel-edit-authentication-button` }
+                                >
+                                    { t("console:manage.features.actions.buttons.cancel") }
+                                </Button>
+                            ) }
+                        </div>
+                    </Box>
+                );
+            };
+
+            return ( !isAuthenticationUpdateFormState && !isCreateFormState && !(authenticationType === null) ?
+                renderAuthenticationSectionInfoBox() : renderAuthenticationSectionUpdatingBox());
+        };
+
+        return isLoading ? renderLoadingPlaceholders() : (
```

Comment:
Better to avoid ternary.

```
if (isLoading) {
    renderLoadingPlaceholders()
}

return (
   // rest
)
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847